Backend/app/dashboard/CommonAPIs/spec/ProviderPlatform/Management/API/Account.yaml

```diff
@@ -32,6 +32,9 @@ apis:
             auth: ApiAuthV2
             request:
               type: FleetOwnerRegisterReq
+            query:
+              - fleetOwnerId: Id Person
```

Comment:
```
mandatoryQuery:
- fleetOwnerId: Id Person
- requireAdminApprovalForFleetOnboarding: Bool
```

Reply:
I think this param not required 
`- requireAdminApprovalForFleetOnboarding: Bool`
-because we already received Approve/Reject from admin